packages/round-manager/src/features/round/ApplicationFormValidationSchema.tsx

```diff
@@ -0,0 +1,71 @@
+import * as yup from "yup";
```

Comment:
this file is not a react component, the name needs to be camel case and extension `.ts`